viadot/flows/sql_server_transform.py

```diff
@@ -0,0 +1,37 @@
+from prefect import Flow, config
+from typing import Any, Dict, List, Literal
+
+from ..tasks import SQLServerQuery
+
+query_task = SQLServerQuery()
+
+
+class SQLServerTransform(Flow):
+    def __init__(
+        self,
+        name: str,
+        query: str,
+        config_key: dict = None,
```

Comment:
As discussed, this will not work when config_key is not provided. Please fix this issue.